CHANGELOG.md

```diff
@@ -1,6 +1,53 @@
 # go-ipfs changelog
 
-## 0.4.16
+## 0.4.17  2018-07-20
+
+Ipfs 0.4.17 is a quick release to fix a major performance regression in bitswap.
```

Comment:
Worth noting that the regression only affects go to javascript transfers

Reply:
Affects all, just mostly JS.